df.world-site.xml

```diff
@@ -58,7 +58,7 @@
 
         <stl-vector name='inhabitants'>
             <pointer>
-                <int16_t/>
+                <int16_t ref-target='historical_figure'/>
                 <int32_t ref-target='historical_figure'/>
```

Comment:
Is it possible that this is actually 2 int32_t's with one set to 0 sometimes? I haven't investigated, but a 16-bit histfig ID followed by a 32-bit histfig ID seems odd to me.